dll/cpl/access/general.c

```diff
@@ -194,8 +196,9 @@ GeneralPageProc(HWND hwndDlg,
                            IDC_RESET_BOX,
                            pGlobalData->accessTimeout.dwFlags & ATF_TIMEOUTON ? BST_CHECKED : BST_UNCHECKED);
             FillResetComboBox(GetDlgItem(hwndDlg, IDC_RESET_COMBO));
-            SendDlgItemMessage(hwndDlg, IDC_RESET_COMBO, CB_SETCURSEL,
-                               (pGlobalData->accessTimeout.iTimeOutMSec / 300000) - 1, 0);
+            pGlobalData->accessTimeout.iTimeOutMSec = max(FIVE_MINS_IN_MS, pGlobalData->accessTimeout.iTimeOutMSec);
+            iCurSel = (pGlobalData->accessTimeout.iTimeOutMSec / FIVE_MINS_IN_MS) - 1;
```

Comment:
```suggestion
            iCurSel = pGlobalData->accessTimeout.iTimeOutMSec >= FIVE_MINS_IN_MS
                      ? (pGlobalData->accessTimeout.iTimeOutMSec / FIVE_MINS_IN_MS) - 1 : 0;
```

Reply:
What would that fix? Imho the existing code is easier to read for humans.

Reply:
Modifying `iTimeOutMSec` for no reason. (I'm not sure which "existing" code you refer to.)

Reply:
With "the existing code" I did mean the code in this PR before your suggestion.
I can at least better understand the motivation behind your suggestion a bit better now. Neutrality therefore from my side for this specific proposal.